.github/dependabot.yml

```diff
@@ -13,6 +13,32 @@ updates:
       interval: "weekly"
       day: "thursday"
     open-pull-requests-limit: 50
+    groups:
+      storybook:
+        patterns:
+          - "@storybook*"
```

Comment:
I think we will also need `storybook` here for the main package that is not namespaced behind the org

Reply:
Yes, yes you are right
https://github.com/GCTC-NTGC/gc-digital-talent/pull/9481/commits/954a1e0ce425c1a36b99b7edc26d26f2c0481811